### PR TITLE
Hb cloud events

### PR DESCRIPTION
**Pull request by HaroldBernard**

### CODE REVIEW

**Comment by HaroldBernard** [comment]
query dev

File: handler/handler.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+var _ events.UserEventsHandler = &UserEventsHandler{}
+
+type UserEventsHandler struct {
+	events.NoopUserEventsHandler
+	MarketoManager marketo.Manager
+}
+
+func (u *UserEventsHandler) HandleCreateUserEvent(event events.CreateUserEvent) error {
+	log.Printf("Received create user event: %v", event)
+	u.MarketoManager.CreateListMembershipForUser(event.UserID, event.UserData)
```
Comment (toddkazakov) [inline handler/handler.go:18]: return an error if the manager fails

File: handler/handler.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	return nil
+}
+
+func (u *UserEventsHandler) HandleUpdateUserEvent(event events.UpdateUserEvent) error {
+	log.Printf("Received update user event: %v", event)
+	u.MarketoManager.UpdateListMembershipForUser(event.Updated.UserID, event.Updated, false)
```
Comment (toddkazakov) [inline handler/handler.go:24]: return an error if the manager fails

File: handler/handler.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	return nil
+}
+
+func (u *UserEventsHandler) HandleDeleteUserEvent(event events.DeleteUserEvent) error {
+	log.Printf("Received delete user event: %v", event)
+	u.MarketoManager.UpdateListMembershipForUser(event.UserID, event.UserData, true)
```
Comment (toddkazakov) [inline handler/handler.go:30]: return an error if the manager fails

**Pull request merged**